community/en/tiny_imagenet_custom_resnet.ipynb

```diff
@@ -0,0 +1,552 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Tce3stUlHN0L"
+   },
+   "source": [
+    "##### Copyright 2019 The TensorFlow Authors.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "tuOe1ymfHZPu"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "# https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "MfBg1C5NB3X0"
+   },
+   "source": [
+    "# TinyImageNet Custom ResNet\n",
+    "\n",
+    "\n",
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/examples/blob/master/template/notebook.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/examples/blob/master/template/notebook.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "xHxb-dlhMIzW"
+   },
+   "source": [
+    "## Overview\n",
+    "{TODO: Fill in detailed info of what this accomplishes}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip install -q tensorflow==2.0.0-beta1\n",
+    "import tensorflow as tf\n",
+    "assert tf.__version__.startswith('2')\n",
+    "\n",
+    "print(f'{tf.__version__}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "download_path = os.getcwd()\n",
+    "    \n",
+    "path = tf.keras.utils.get_file('tiny-imagenet-200.zip', extract=True, cache_subdir=download_path,\n",
```

Comment:
Notebook currently fails, I think you want:

```
import os
download_path = os.getcwd()

import pathlib    
zip_path = tf.keras.utils.get_file('tiny-imagenet-200.zip', extract=True,
                               origin='http://cs231n.stanford.edu/tiny-imagenet-200.zip')

data_dir = pathlib.Path(zip_path).with_suffix('')

TRAIN = data_dir/"train"
VAL = data_dir/"val"
VAL_ANNOT = data_dir/'val/val_annotations.txt'
```

Reply:
Yes, I have fixed this. It will reflect in the next commit. I had not updated the tested code from training on multiple colab accounts.

Reply:
One of the 12 models I'm simultaneously training on colab converged after 356 epochs with SGD and Cyclic Learning Rate. That will be difficult to capture here, so am thinking of some other possibility using CLR rather than show the convergence.